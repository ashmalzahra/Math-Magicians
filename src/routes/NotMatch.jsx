const NotMatch = () => (
  <p>Page not found</p>
);

export default NotMatch;
