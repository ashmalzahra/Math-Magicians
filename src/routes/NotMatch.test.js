import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import NotMatch from './NotMatch';
import '@testing-library/jest-dom';

describe('Checking page deployment', () => {
  test('NotMatch component matches snapshot', () => {
    const tree = renderer.create(<NotMatch />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders NotMatch component', () => {
    render(<NotMatch />);

    expect(screen.getByText(/Page not found/)).toBeInTheDocument();
  });
});
