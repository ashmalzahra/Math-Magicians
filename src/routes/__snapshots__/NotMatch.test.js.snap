// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking page deployment NotMatch component matches snapshot 1`] = `
<p>
  Page not found
</p>
`;
